src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -1206,13 +1206,13 @@ private static void WriteSimpleValue(bool indented, string jsonIn, string jsonOu
         private static void AssertContents(string expectedValue, ArrayBufferWriter<byte> buffer)
         {
             Assert.Equal(
-                expectedValue,
+                expectedValue.NormalizeLineEndings(),
                 Encoding.UTF8.GetString(
                     buffer.WrittenSpan
 #if netfx
                         .ToArray()
 #endif
-                    ));
+                    ).NormalizeLineEndings());
```

Comment:
Couldn't this end up hiding bugs where, for example, the writer ends up using "\n" on Windows?

Reply:
I'll create a second copy of AssertContent which normalizes and verify all callsites - this way we will avoid any potential issues now and in the future (name should be clearer what it does)

Reply:
ended up doing it directly on each test as it was more clearer what's going on